components/eamxx/src/control/atmosphere_surface_coupling_exporter.cpp

```diff
@@ -436,13 +437,13 @@ void SurfaceCouplingExporter::compute_eamxx_exports(const double dt, const bool
       // Precipitation has units of kg/m2, and Faxa_rainl/snowl
       // need units mm/s. Here, 1000 converts m->mm, dt has units s, and
       // rho_h2o has units kg/m3.
-      if (m_export_source(idx_Faxa_rainl)==FROM_MODEL) { Faxa_rainl(i) = precip_liq_surf_mass(i)/dt*(1000.0/PC::RHO_H2O); }
-      if (m_export_source(idx_Faxa_snowl)==FROM_MODEL) { Faxa_snowl(i) = precip_ice_surf_mass(i)/dt*(1000.0/PC::RHO_H2O); }
+      if (export_source(idx_Faxa_rainl)==FROM_MODEL) { Faxa_rainl(i) = precip_liq_surf_mass(i)/dt*(1000.0/PC::RHO_H2O); }
+      if (export_source(idx_Faxa_snowl)==FROM_MODEL) { Faxa_snowl(i) = precip_ice_surf_mass(i)/dt*(1000.0/PC::RHO_H2O); }
     }
   });
   // Variables that are already surface vars in the ATM can just be copied directly.
-  auto export_source_h = Kokkos::create_mirror_view(m_export_source);
-  Kokkos::deep_copy(export_source_h,m_export_source);
+  auto export_source_h = Kokkos::create_mirror_view(export_source);
```

Comment:
You might as well save a host copy of export source in the class, so you don't need to allocate/copy every timestep..